documentation/modules/ROOT/pages/connectors/mysql.adoc

```diff
@@ -2200,6 +2201,62 @@ Contains a structure with two fields:
 
 |===
 
+[[debezium-mysql-connector-converters]]
+== Custom converters
+
+The {prodname} MySQL connector provides several `CustomConverter` implementations specific for MySQL out of the box.
+These custom converters provide alternative mappings for specific data types influenced by configuration.
+To add a `CustomConverter` to the connector, follow the instructions in the link:../development/converters.adoc[Custom Converters documentation].
+
+=== `TINYINT(1)` to Boolean
+
+MySQL reports `BOOLEAN` data types as `TINYINT(1)` during the connector's snapshot.
+This may not be ideal as it will create a difference in the change event schema with the streaming phase.
+To make sure that MySQL emits `BOOLEAN` columns uniformly, the `TinyIntOneToBooleanConverter` can be used.
```

Comment:
```suggestion
To help ensure that MySQL emits `BOOLEAN` columns uniformly, you can apply the custom  `TinyIntOneToBooleanConverter`, as shown in the following configuration example.
```
In general, the lawyers cringe if the docs imply that anything is `sure` or `guaranteed`. But we can _help_ in that regard.